src/st/cen/cen.h

```diff
@@ -85,6 +85,7 @@ extern ObjInit2 D_8018125C[];
 
 extern LayoutEntity* D_8019C764;
 extern u16* D_8019C768;
+#define g_LayoutObjEnd ((LayoutEntity*)D_8019C768)
```

Comment:
I suggest you to just do `g_LayoutObjEnd = 0x8019C768` in `config/symbols.us.stcen.txt`. It reflects on `build/us/stcen.map` too. Using a `#define` will hide things a bit too much.

Reply:
I was going to save that for a refactor but it turns out the refactor will probably capture this change anyway.

Nearly all of the layout scanning and creation functions are identical, they just need symbols mapped. That's probably a better change that this one off. I'm going to try to get that finished today and post a PR, but it's a much broader change across all of the stages.

Reply:
It will be fine to just have the symbol added in stcen for this PR and have the refactor as a separate one if you want to avoid having too many changes in one go